### PR TITLE
Fix resources section link

### PR DESCRIPTION
**Pull request by bhaprayan**
Updated the resources section link in the README file to point to the correct page on tensorflow.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,4 +67,4 @@ Hello, TensorFlow!
 * [TensorFlow Model Zoo](https://github.com/tensorflow/models)
 * [TensorFlow MOOC on Udacity](https://www.udacity.com/course/deep-learning--ud730)
 
-The TensorFlow community has created amazing things with TensorFlow, please see the [resources section of tensorflow.org](https://www.tensorflow.org/versions/master/resources#community) for an incomplete list.
+The TensorFlow community has created amazing things with TensorFlow, please see the [resources section of tensorflow.org](https://www.tensorflow.org/about/) for an incomplete list.
```
Comment (caisq) [inline README.md]: Please use the URL with the bookmark: https://www.tensorflow.org/about/#community

**Review by caisq — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bhaprayan) [inline README.md]: Sure. Done!

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
PR merged. Thanks, @bhaprayan !

**Comment by bhaprayan** [comment]
Sure! @caisq

**Pull request merged**